chain/client/src/metrics.rs

```diff
@@ -361,12 +361,12 @@ pub(crate) static CURRENT_PROTOCOL_VERSION: Lazy<IntGauge> = Lazy::new(|| {
         .unwrap()
 });
 
-pub(crate) static NODE_PROTOCOL_UPGRADE_VOTING_START: Lazy<IntGauge> = Lazy::new(|| {
-    try_create_int_gauge(
-        "near_node_protocol_upgrade_voting_start",
-        "Time in seconds since Unix epoch determining when node will start voting for the protocol upgrade; zero if there is no schedule for the voting")
-        .unwrap()
-});
+// pub(crate) static NODE_PROTOCOL_UPGRADE_VOTING_START: Lazy<IntGauge> = Lazy::new(|| {
+//     try_create_int_gauge(
+//         "near_node_protocol_upgrade_voting_start",
+//         "Time in seconds since Unix epoch determining when node will start voting for the protocol upgrade; zero if there is no schedule for the voting")
+//         .unwrap()
+// });
```

Comment:
Please ignore for now, I'll adjust this metric to show the voting start time per protocol version.